charts/ncbiaccess/templates/imagePullSecret.yaml

```diff
@@ -0,0 +1,15 @@
+apiVersion: secrets-manager.tuenti.io/v1alpha1
+kind: SecretDefinition
+metadata:
+  name: {{ .Values.name }}-dockerhub-creds
+  labels:
+{{ include "ncbiaccess.labels" . | indent 4 }}
+spec:
+  name: {{ .Values.name }}-dockerhub-creds
+  type: kubernetes.io/dockerconfigjson
```

Comment:
This is really interesting, props for figuring it out!

Reply:
NP. I'm going to do a write up in our gitbook of what is actually needed in order to pull from private repos. There are definitely some gotchas and conflicting information out there.